actionizer.gemspec

```diff
@@ -25,13 +25,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '~> 2.1'
+  spec.required_ruby_version = '~> 3.3'
```

Comment:
I think you could make this `spec.required_ruby_version = '>= 2.1', '< 4'` and basically undo all other changes